portfolio/src/main/webapp/blog.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Chris Elliott</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Chris Elliott</h1>
+      <div id="profile-links">
+        <a class="profile-link" href="https://github.com/zpChris" target="_blank">GitHub</a>
+        <a class="profile-link" href="https://www.linkedin.com/in/chris-m-elliott/" target="_blank">LinkedIn</a>
+        <a class="profile-link" href="/">Homepage</a>
+      </div>
+      <h2>Personal Blog</h2>
+      <div class="blog-post">
+        <h3>Summer 2020</h3>
+        <h4>May 20th, 2020</h4>
+        <p>
+            This summer, outside of Google, I am hoping to:
+            <ol>
+                <li>Explore running (long-distance) and calisthenics</li>
+                <li>Work on <a href="https://lambdaai.dev" target="_blank">Lambda AI</a>, and formally release the product</li>
+                <li>Read more (just read "The Happiness Advantage," now I'm reading "How We Learn")</li>
+            </ol>
+        </p>
+        <div class="gallery-container">
+            <div class="gallery-image-container">
+                <img class="gallery-image" src="images/running.jpg"/>
```

Comment:
It would be nice to add an alt attribute to these images. Alt attributes are usually picked up by screenreaders used by blind / vision impaired users.

More info in the google html/css style guide here: https://engdoc.corp.google.com/eng/doc/devguide/htmlcss/styleguide.md?cl=head#Multimedia_Fallback

Reply:
Ok, sounds good!

Also, I noted this to Bootsie, but I'm not able to view style guides like the one you linked above, or ones at `go/htmlcssstyle`. I asked my teammates about this, and Eshika cannot view these either; they are unfortunately restricted access.

Reply:
hmm, ok I'll follow up to see if we can get access for interns. In the mean time here's the public version which is mostly identical: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback